### PR TITLE
Tests for weight and power calculations

### PR DESCRIPTION
**Pull request by anorth**

### CODE REVIEW

**Comment by anorth** [comment]
FYI @davidad no abnormalities found here. I'll try to repro the unexpected power I found earlier another way.

File: actors/builtin/miner/policy_test.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+		fullyVerifiedPower := big.Div(big.Mul(big.NewInt(int64(sectorSize)), builtin.VerifiedDealWeightMultiplier), builtin.QualityBaseMultiplier)
+
+		maxError := big.NewInt(1 << builtin.SectorQualityPrecision)
+
+		for _, verifiedSpace := range []abi.SectorSize{0, 1, 1 << 10, 5 << 20, 32 << 30, sectorSize - 1, sectorSize} {
+			for _, verifiedDuration := range []abi.ChainEpoch{0, 1, sectorDuration / 2, sectorDuration - 1, sectorDuration} {
+				verifiedWeight := weight(verifiedSpace, verifiedDuration)
+				emptyWeight := big.Sub(sectorWeight, verifiedWeight)
+
+				// Expect sector power to be a proportional sum of base and verified power.
```
Comment (acruikshank) [inline actors/builtin/miner/policy_test.go:89]:
```suggestion
				// Expect sector power to be proportional to sum of base and verified power.
```

File: actors/builtin/miner/policy_test.go [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+		sectorSize := abi.SectorSize(1 << 40)
+		sectorDuration := abi.ChainEpoch(1_000_000) // ~350 days
+		sectorWeight := weight(sectorSize, sectorDuration)
+
+		for _, verifiedSpace := range []abi.SectorSize{0, 1, 1 << 10, 2 << 20, 5 << 20, 1 << 30, sectorSize - 1, sectorSize} {
+			emptyWeight := weight(sectorSize-verifiedSpace, sectorDuration)
+			verifiedWeight := weight(verifiedSpace, sectorDuration)
+			assertEqual(t, big.Sub(sectorWeight, emptyWeight), verifiedWeight)
+
+			// Expect sector quality to be a proportional sum of base and verified quality.
```
Comment (acruikshank) [inline actors/builtin/miner/policy_test.go:43]:
```suggestion
			// Expect sector quality to be proportional to sum of base and verified quality.
```

File: actors/builtin/miner/policy_test.go (hunk 1 above)
Comment (anorth) [reply actors/builtin/miner/policy_test.go:89]: Changed to "weighed sum"

**Pull request merged**